index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+
+<head>
+  <title> </title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <main>
```

Comment:
Looking at your wireframe, it seems like this main tag should actually be a header tag! Your main tag should be used for the main body of your website while your header should include images and your nav.